pkgs/top-level/all-packages.nix

```diff
@@ -15103,7 +15103,7 @@ with pkgs;
 
   mkcert = callPackage ../development/tools/misc/mkcert { };
 
-  mkdocs = callPackage ../development/tools/documentation/mkdocs { };
+  mkdocs = with python3Packages; toPythonApplication mkdocs;
```

Comment:
Should've used `git mv` for the derivation. Now `git rm` should pick up the move.